packages/flutter/lib/src/material/text_field.dart

```diff
@@ -423,7 +423,13 @@ class TextField extends StatefulWidget {
   ///
   /// This text style is also used as the base style for the [decoration].
   ///
-  /// If null, defaults to the `titleMedium` text style from the current [Theme].
+  /// If null and [ThemeData.useMaterial3] is true, [TextTheme.bodyLarge] will
```

Comment:
nit: since the default `useMaterial3` is already true, I think maybe we don't need to explicitly mention it here. But either way looks good to me:)

Reply:
That's a good point. I updated the docs.